base/applications/taskmgr/lang/pl-PL.rc

```diff
@@ -277,40 +283,40 @@ BEGIN
     AUTOCHECKBOX "CPU 31", IDC_CPU31, 175, 119, 41, 10, WS_DISABLED | WS_TABSTOP
 END
 
-IDD_COLUMNS_DIALOG DIALOGEX 0, 0, 195, 199
+IDD_COLUMNS_DIALOG DIALOGEX 0, 0, 210, 199
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Wybierz kolumny"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    DEFPUSHBUTTON "OK", IDOK, 84, 178, 50, 14
-    PUSHBUTTON "Anuluj", IDCANCEL, 138, 178, 50, 14
+    DEFPUSHBUTTON "OK", IDOK, 94, 178, 50, 14
+    PUSHBUTTON "Anuluj", IDCANCEL, 148, 178, 50, 14
     LTEXT "Wybierz kolumny do wyświetlania na stronie Proces Menedżera zadań", IDC_STATIC, 7, 7, 181, 17
-    AUTOCHECKBOX "&Nazwa obrazu", IDC_IMAGENAME, 7, 28, 100, 10, WS_DISABLED | WS_TABSTOP
-    AUTOCHECKBOX "PI&D (idewntyfikator procesu)", IDC_PID, 7, 39, 100, 10
-    AUTOCHECKBOX "&Użycie procesora CPU", IDC_CPUUSAGE, 7, 50, 100, 10
-    AUTOCHECKBOX "&Czas procesora CPU", IDC_CPUTIME, 7, 61, 100, 10
-    AUTOCHECKBOX "Zużycie pa&mięci", IDC_MEMORYUSAGE, 7, 72, 100, 10
-    AUTOCHECKBOX "&Zmiana zużycia pamięci", IDC_MEMORYUSAGEDELTA, 7, 83, 100, 10
-    AUTOCHECKBOX "&Szczytowe zużycie pamięci", IDC_PEAKMEMORYUSAGE, 7, 94, 100, 10
-    AUTOCHECKBOX "&Błędy stronic", IDC_PAGEFAULTS, 7, 105, 100, 10
-    AUTOCHECKBOX "Obie&kty USER", IDC_USEROBJECTS, 7, 116, 100, 10
-    AUTOCHECKBOX "Odczyty We/Wy", IDC_IOREADS, 7, 127, 100, 10
-    AUTOCHECKBOX "Odczyty We/Wy w bajtach", IDC_IOREADBYTES, 7, 138, 100, 10
-    AUTOCHECKBOX "&Identyfikator sesji", IDC_SESSIONID, 7, 149, 100, 10
-    AUTOCHECKBOX "Naz&wa użytkownika", IDC_USERNAME, 7, 160, 100, 10
-    AUTOCHECKBOX "Zmi&ana błędów stronic", IDC_PAGEFAULTSDELTA, 107, 28, 85, 10
-    AUTOCHECKBOX "&Rozmiar pamięci wirtualnej", IDC_VIRTUALMEMORYSIZE, 107, 39, 85, 10
-    AUTOCHECKBOX "&Pula stronicowania", IDC_PAGEDPOOL, 107, 50, 85, 10
-    AUTOCHECKBOX "Pula niestr&onicowana", IDC_NONPAGEDPOOL, 107, 61, 85, 10
-    AUTOCHECKBOX "Prior&ytet podstawowy", IDC_BASEPRIORITY, 107, 72, 85, 10
-    AUTOCHECKBOX "&Liczba uchwytów", IDC_HANDLECOUNT, 107, 83, 85, 10
-    AUTOCHECKBOX "Liczba w&ątków", IDC_THREADCOUNT, 107, 94, 85, 10
-    AUTOCHECKBOX "Obiekty &GDI", IDC_GDIOBJECTS, 107, 105, 85, 10
-    AUTOCHECKBOX "Zapisy We/Wy", IDC_IOWRITES, 107, 116, 85, 10
-    AUTOCHECKBOX "Zapisy We/Wy w bajtach", IDC_IOWRITEBYTES, 107, 127, 85, 10
-    AUTOCHECKBOX "Inne We/Wy", IDC_IOOTHER, 107, 138, 85, 10
-    AUTOCHECKBOX "Inne We/Wy w bajtach", IDC_IOOTHERBYTES, 107, 149, 85, 10
-    AUTOCHECKBOX "Linia poleceń", IDC_COMMANDLINE, 107, 160, 85, 10
+    AUTOCHECKBOX "&Nazwa obrazu", IDC_IMAGENAME, 7, 28, 99, 10, WS_DISABLED | WS_TABSTOP
```

Comment:
Is it possible to uniformize a bit more the width of these autocheckboxes? like all, using 100, or all using 98 ?